js/src/sdk/index.spec.ts

```diff
@@ -26,54 +26,70 @@
   });
 
   it("should handle 404 error gracefully", async () => {
-    const client = new Composio({ apiKey: COMPOSIO_API_KEY });
     const mock = new AxiosMockAdapter(axiosClient.instance);
-    mock.onGet("/api/v1/apps").reply(404, { detail: "Not found" });
+    const mockError = {
+      error: {
+        type: "NotFoundError",
+        name: "AppNotFoundError",
+        message: "Not found",
+      },
+    };
+    mock.onGet("/api/v1/apps").reply(404, mockError);
+
+    const client = new Composio({ apiKey: COMPOSIO_API_KEY });
 
     try {
-      await client.apps.list();
+      const apps = await client.apps.list();
+      console.log(apps);
     } catch (e) {
-      const error = e as ComposioError;
-      const errorCode = COMPOSIO_SDK_ERROR_CODES.BACKEND.NOT_FOUND;
-      const errorInfo = BASE_ERROR_CODE_INFO[errorCode];
-      expect(error.errCode).toBe(errorCode);
-      expect(error.message).toContain(errorInfo.message);
-      expect(error.description).toBe(errorInfo.description);
-      expect(error.errorId).toBeDefined();
-      expect(error.name).toBe("ComposioError");
-      expect(error.possibleFix).toBe(errorInfo.possibleFix);
+      if (e instanceof ComposioError) {
+        console.log(e);
```

Comment:
Remove `console.log(e)` from the test case. Debug logs should not be committed to the test suite.